site/content/docs/commands/env-package.en.md

```diff
@@ -0,0 +1,30 @@
+# env package
+```console
+$ copilot env package [flags]
+```
+
+## What does it do?
+`copilot env package` prints the CloudFormation stack template and configuration used to deploy an environment.
+
+## What are the flags?
+```console
+-a, --app string          Name of the application.
```

Comment:
🤔 Do we need to line these up or does mkdocs magically do it? Other files appear to be lined up...?

Reply:
Done!